zerver/lib/exceptions.py

```diff
@@ -44,6 +44,7 @@ class ErrorCode(AbstractEnum):
     INVALID_MARKDOWN_INCLUDE_STATEMENT = ()
     REQUEST_CONFUSING_VAR = ()
     INVALID_API_KEY = ()
+    UNSUCCESSFUL_RESPONSE_ERROR = ()
```

Comment:
Oops, sorry for not pointing this out earlier, but this should probably be changed to something more informative such as `THIRD_PART_API_RESPONSE_ERROR` :)

Reply:
Oops, I'm sorry I missed that. It's not your fault, I should have been consistent with the change.

Reply:
Made the change.